### DISCUSSION TITLE
docs: Add example with forwardRef in Icon and Tooltip

### DISCUSSION THREAD

**Discussion by evgeniyworkbel** [ant-design/ant-design#41510] [Ideas]
I have opened [issue](https://github.com/ant-design/ant-design/issues/41497) "findDOMNode is deprecated" and one contributor [suggested](https://codesandbox.io/s/ji-ben-antd-5-3-2-forked-sfx3oy?file=/demo.tsx) to use forwardRed when I wrap my svg into Icon component.

Please add example with forwardRef in Icon and Tooltip. Thus other devs can wrap their svg right way

**Comment by zombieJ** [comment]
Thanks for feedback. But `forwardRef` is the [base usage](https://react.dev/reference/react/forwardRef) of React and nothing magic on antd. As the component lib, it's not a good place to guide how to use `forwardRef` since if React official modify its usage or provide new method, third part doc will be OOD of this.

For other common asked question, we also link to React official site in FAQ instead of duplicated intro:
https://ant.design/docs/react/faq
